.github/workflows/compilation_on_sgx.yml

```diff
@@ -322,11 +322,18 @@ jobs:
       matrix:
         running_mode: ["classic-interp", "fast-interp", "aot"]
         test_option: ["-x -p -s spec -P", "-x -p -s spec -S -P"]
+        exclude:
+          - running_mode: "classic-interp"
```

Comment:
Had better add comments, here same as other file: `# classic-interp and fast-interp don't support simd`